addons/website_event_exhibitor/report/website_event_exhibitor_templates.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <template id="event_report_full_page_ticket_layout_inherit_exhibitor" inherit_id="event.event_report_full_page_ticket_layout">
-        <xpath expr="//div[hasclass('o_event_full_page_ticket_powered_by')]" position="before">
+    <template id="event_report_template_full_page_ticket_inherit_exhibitor" inherit_id="event.event_report_template_full_page_ticket">
+        <xpath expr="//div[hasclass('o_event_full_page_ticket_details')]" position="inside">
             <div t-if="not responsive_html" class="o_event_full_page_ticket_sponsors_container text-center mb-2">
-                <t t-foreach="event.sponsor_ids.filtered(lambda sponsor: sponsor.image_128 and sponsor.website_published)[:10]"
-                    t-as="sponsor">
+                <t t-set="sponsors" t-value="event.sponsor_ids.filtered(
```

Comment:
This is no longer limited to the first 10 sponsors?

How does it behave if there are 20 sponsors? Or 100?

Reply:
Ok but it would be good to keep some kind of safeguard to avoid the ticket report from looking bad, we put that number there for a reason. 

On another note:
Are we sure we want to hide them by default?
It's the other way around currently.
Which means that when people migrate to this version, suddenly their ticket reports show no sponsors anymore.

Might be interesting to re-discuss this point with JIJA to make sure it's ok.